src/PKSim.Core/Services/IndividualTransporterFactory.cs

```diff
@@ -84,6 +84,9 @@ public IndividualTransporter UndefinedLiverTransporterFor(Individual individual)
             );
          });
 
+         _individualPathWithRootExpander.AddRootToPathIn(individual, transporter.Name);
```

Comment:
@Yuri05 This was missing here as well as for Enzyme. Also I am  now adding the molecule directlyhere instead of letting the caller having to do it. IT was ok before, it is not now